media-gfx/yafaray/yafaray-3.5.1.ebuild

```diff
@@ -0,0 +1,109 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{7,8,9} )
```

Comment:
Isn't 3_7 already getting removed?

Reply:
For now, let's keep it, until it becomes a problem.

Reply:
Usually they also get removed in one big push. If this were a new package or the python dep would be an addition to the previous version, I wouldn't add python-3.7.